pkg/provisioner/ironic/ironic.go

```diff
@@ -544,18 +549,18 @@ func (p *ironicProvisioner) ValidateManagementAccess(credentialsChanged, force b
 		return
 
 	case nodes.Manageable:
-		p.log.Info("have manageable host")
+		p.debugLog.Info("have manageable host")
```

Comment:
Could probably just delete this one, given that it is redundant with the one on line 514.